data/projects/coala_cli.md

```diff
@@ -19,32 +19,84 @@ project is going to rely heavily upon mockups and feedback from the
 community. Also, in addition to the above changes the applicant is encouraged
 to come up with their own ideas to improve the CLI.
 
```

Comment:
imo it would be a good idea to split those that are in progress from those that are done, in the list below. let's say you use italic or so for those WIP, such as:
 - Implemented x, y
 - *Design the new algorithm*

NOTE: Those written in italic are Work in Progress.